NitroxModel/Helper/WebHelper.cs

```diff
@@ -18,15 +20,15 @@ public static string GetPublicIP()
             }
         }
 
-        public static string GetNitroxLatestVersion()
+        public static Version GetNitroxLatestVersion()
         {
             HttpWebRequest req = WebRequest.Create("https://api.github.com/repos/SubnauticaNitrox/Nitrox/releases/latest") as HttpWebRequest;
             req.Method = "GET";
             req.UserAgent = "Nitrox";
             req.ContentType = "application/json";
             req.Timeout = 1000;
```

Comment:
Only 1 second seems on the short side.
```suggestion
            req.Timeout = 5000;
```